src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -41,7 +41,9 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
 
   return (
     <div className={posterClassnames} onClick={handleEnterVenue}>
-      <p className="PosterPreview__title">{title}</p>
+      <p className="PosterPreview__title">
+        {posterVenue.name}: {title}
+      </p>
```

Comment:
@sparkletown/product @mike-lvov Would you be happy with this change being made (to show the `posterVenue` name in full + the poster title) instead of https://github.com/sparkletown/internal-sparkle-issues/issues/755 (which isn't implementable in a generalisable way as we don't (to my knowledge) have a concept of 'poster number' outside of the implied form of it through `posterVenue.name`)

The changes in this file represent the `PosterPreview` 'cards' in the `PosterHall`.

And if so, would you be happy for that to be a change made/merged into our core codebase? Or would we do this as an `env/ohbm` specific hack?